.github/workflows/update.yaml

```diff
@@ -32,6 +32,7 @@ jobs:
           ${GITHUB_WORKSPACE}/.tmp/CODEOWNERS \
           ${GITHUB_WORKSPACE}/.tmp/.pre-commit-config.yaml \
           ${GITHUB_WORKSPACE}/.tmp/.gitignore \
+          ${GITHUB_WORKSPACE}/.tmp/PULL_REQUEST_TEMPLATE.md \
```

Comment:
Instead of moving individual files should we just copy the entire content? This means when you add a new file you don't have to edit here.

Reply:
Copying over the entire content would replace other files e.g main.tf files and READMEs with the ones in `terraform-module-template`..hence break the module code.

Reply:
Ok I'm approving this but we have to rethink the distribution strategy then.